proposals/nnnn-app-services.md

```diff
@@ -509,7 +509,18 @@ When an app service consumer receives an `AppServiceData` object that contains f
 
 #### App Service Actions - `PerformAppServiceInteraction`
 
-Talk about how this works here.
+App service providers will likely have different actions exposed to the module and app service consumers. It will be difficult to standardize these actions by RPC versions and can easily become stale. Therefore, we introduce a best-effort attempt to take actions on a service. 
+
+The `PerformAppServiceInteraction` request be sent to the service that has the matching `appServiceId`. the `serviceUri` should be the fully qualified URI with all parameters that are necessary for the given action. The URI prefix and actions list will be contained in the app service provider's manifest. 
```

Comment:
The first sentence should be updated to "...request will be sent..." or "...request should be sent..."  Please also capitalize "The" in the second sentence.